sync.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+import os
+import logging
+from oc_art_to_ftp.app.art_to_ftp import ArtToFTP
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Please do not hardcode log level. Use that set in the web server, or add as an argument.